h2gis-functions/src/test/java/org/h2gis/functions/spatial/metadata/GeometryTableUtilsTest.java

```diff
@@ -606,5 +607,74 @@ public void testGetMetadatasFromResulset() throws SQLException {
+        assertEquals("GEOMETRY", geomMetadata.getSQL());       
+        st.execute("ALTER TABLE GEO_POINT ALTER COLUMN THE_GEOM type geometry(POINT Z, 4326)");
+        geomMetadata = GeometryTableUtilities.getMetaData(connection, TableLocation.parse("GEO_POINT"), "THE_GEOM");
+        assertEquals("GEOMETRY(POINTZ,4326)", geomMetadata.getSQL());        
+        st.execute("ALTER TABLE GEO_POINT ALTER COLUMN THE_GEOM type geometry(POINTZM)");
+        geomMetadata = GeometryTableUtilities.getMetaData(connection, TableLocation.parse("GEO_POINT"), "THE_GEOM");
+        assertEquals("GEOMETRY(POINTZM)", geomMetadata.getSQL());        
+    }
+    
+    @Test
+    public void testAlterSRID() throws Exception {
+        st.execute("drop table if exists geo_point; CREATE TABLE geo_point (the_geom GEOMETRY)");
+        st.execute("insert into geo_point VALUES('SRID=0;POINT(0, 0, 0)'");
+        GeometryMetaData geomMetadata = GeometryTableUtilities.getMetaData(connection, TableLocation.parse("GEO_POINT"), "THE_GEOM");
+        assertEquals(0, geomMetadata.getSRID());       
+        GeometryTableUtilities.alterSRID(connection, TableLocation.parse("GEO_POINT"), "the_geom", 4326);
+        geomMetadata = GeometryTableUtilities.getMetaData(connection, TableLocation.parse("GEO_POINT"), "THE_GEOM");
+        assertEquals("GEOMETRY(GEOMETRY,4326)", geomMetadata.getSQL());   
+        assertEquals(4326, geomMetadata.getSRID());         
+        st.execute("drop table if exists geo_point; CREATE TABLE geo_point (the_geom GEOMETRY(POINTZ)");
+        geomMetadata = GeometryTableUtilities.getMetaData(connection, TableLocation.parse("GEO_POINT"), "THE_GEOM");
+        assertEquals(0, geomMetadata.getSRID());       
+        GeometryTableUtilities.alterSRID(connection, TableLocation.parse("GEO_POINT"), "the_geom", 4326);
+        geomMetadata = GeometryTableUtilities.getMetaData(connection, TableLocation.parse("GEO_POINT"), "THE_GEOM");
+        assertEquals("GEOMETRY(POINTZ,4326)", geomMetadata.getSQL());   
+        assertEquals(4326, geomMetadata.getSRID());
+        st.execute("drop table if exists geo_point; CREATE TABLE geo_point (the_geom GEOMETRY(POINTZ, 2154)");
+        geomMetadata = GeometryTableUtilities.getMetaData(connection, TableLocation.parse("GEO_POINT"), "THE_GEOM");
+        assertEquals(0, geomMetadata.getSRID());       
+        GeometryTableUtilities.alterSRID(connection, TableLocation.parse("GEO_POINT"), "the_geom", 4326);
+        geomMetadata = GeometryTableUtilities.getMetaData(connection, TableLocation.parse("GEO_POINT"), "THE_GEOM");
+        assertEquals("GEOMETRY(POINTZ,4326)", geomMetadata.getSQL());   
+        assertEquals(4326, geomMetadata.getSRID());
+    }
+    
+    
+    @Test
+    public void testUpdateSRIDFunction() throws Exception {
+        st.execute("drop table if exists geo_point; CREATE TABLE geo_point (the_geom GEOMETRY)");
+        st.execute("insert into geo_point VALUES('SRID=0;POINT(0, 0, 0)'");
```

Comment:
It seems to me that a parenthesis is missing
Shouldn't it be more like this:
`st.execute("insert into geo_point VALUES('SRID=0;POINT(0, 0, 0)')");`
Same on line 627
@ebocher

Reply:
Solved in the master.